test_settings.py

```diff
@@ -16,15 +16,29 @@ def root(*args):
     """
     return join(abspath(dirname(__file__)), *args)
 
+DEBUG = True
+
+ALLOWED_HOSTS = ['testserver', 'localhost', '127.0.0.1']
+
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
Dead code ?

Reply:
Yes, sorry for that. This branch is not intended to be merged. I have pushed it to show you the code!